templates/networkpolicy.go

```diff
@@ -38,3 +38,39 @@ var NetworkPolicyTemplate = netv1.NetworkPolicy{
 		PodSelector: metav1.LabelSelector{},
 	},
 }
+
+var CephNetworkPolicyTemplate = netv1.NetworkPolicy{
+	Spec: netv1.NetworkPolicySpec{
+		Ingress: []netv1.NetworkPolicyIngressRule{
+			{
+				From: []netv1.NetworkPolicyPeer{
+					{
+						NamespaceSelector: &metav1.LabelSelector{
+							MatchLabels: map[string]string{
+								"policy-group.network.openshift.io/host-network": "",
+							},
+						},
+					},
+				},
+			},
+		},
+		PolicyTypes: []netv1.PolicyType{
+			netv1.PolicyTypeIngress,
+		},
+		PodSelector: metav1.LabelSelector{
+			MatchExpressions: []metav1.LabelSelectorRequirement{
+				{
+					Key:      "app",
+					Operator: metav1.LabelSelectorOpIn,
+					Values: []string{
+						"rook-ceph-mds",
+						"rook-ceph-osd",
+						"rook-ceph-mgr",
+						"rook-ceph-mon",
+						"rook-ceph-crashcollector",
+					},
+				},
+			},
+		},
+	},
+}
```

Comment:
Can we please keep each template in its own file

Reply:
Its of same kind, why its really required to have it in a separate file?

Reply:
Made changes accordingly